src/warnet/admin.py

```diff
@@ -33,3 +34,132 @@ def init():
+    ).decode("utf-8")
+
+
+# Get all namespaces that start with "warnet-"
+def get_warnet_namespaces():
+    namespaces = (
+        subprocess.check_output(
+            ["kubectl", "get", "namespaces", "-o", "jsonpath={.items[*].metadata.name}"]
+        )
+        .decode("utf-8")
+        .split()
+    )
+    return [ns for ns in namespaces if ns.startswith("warnet-")]
+
+
+# Get all service accounts for a given namespace
+def get_service_accounts(namespace):
+    return (
+        subprocess.check_output(
+            [
+                "kubectl",
+                "get",
+                "serviceaccounts",
+                "-n",
+                namespace,
+                "-o",
+                "jsonpath={.items[*].metadata.name}",
+            ]
+        )
+        .decode("utf-8")
+        .split()
+    )
+
+
+@admin.command()
+@click.option(
+    "--kubeconfig-dir",
+    default="kubeconfigs",
+    help="Directory to store kubeconfig files (default: kubeconfigs)",
+)
+@click.option(
+    "--token-duration",
+    default=172800,
+    type=int,
+    help="Duration of the token in seconds (default: 48 hours)",
+)
+def create_kubeconfigs(kubeconfig_dir, token_duration):
+    """Create kubeconfig files for all ServiceAccounts in namespaces starting with 'warnet-'."""
+    kubeconfig_dir = os.path.expanduser(kubeconfig_dir)
+
+    cluster_name = get_kubectl_value("{.clusters[0].name}")
+    cluster_server = get_kubectl_value("{.clusters[0].cluster.server}")
+    cluster_ca = get_kubectl_value("{.clusters[0].cluster.certificate-authority-data}")
+
+    os.makedirs(kubeconfig_dir, exist_ok=True)
+
+    # Get all namespaces that start with "warnet-"
+    warnet_namespaces = get_warnet_namespaces()
+
+    for namespace in warnet_namespaces:
+        click.echo(f"Processing namespace: {namespace}")
+        service_accounts = get_service_accounts(namespace)
+
+        for sa in service_accounts:
+            click.echo(f"Processing ServiceAccount: {sa}")
+
+            # Create a token for the ServiceAccount with specified duration
+            try:
+                token = (
+                    subprocess.check_output(
+                        [
+                            "kubectl",
+                            "create",
+                            "token",
+                            sa,
+                            "-n",
+                            namespace,
+                            f"--duration={token_duration}s",
+                        ]
+                    )
+                    .decode("utf-8")
+                    .strip()
+                )
+            except subprocess.CalledProcessError:
+                click.echo(f"Failed to create token for ServiceAccount {sa}. Skipping...")
+                continue
+
+            # Create a kubeconfig file for the user
+            kubeconfig_file = os.path.join(kubeconfig_dir, f"{sa}-{namespace}-kubeconfig")
+
+            kubeconfig_content = f"""apiVersion: v1
```

Comment:
works for now, but ideally we keep the k8s yamls out of the python code. maybe just leave a #TODO comment to move this at some point ?